xonsh/proc.py

```diff
@@ -0,0 +1,30 @@
+"""xonsh.proc is deprecated"""
+from xonsh.lazyasd import lazyobject
+from xonsh.tools import print_warning
+
+
+_WARNINGS_PRINTED = {}
+
+
+def _print_proc_warning(msg):
+    global _WARNINGS_PRINTED
```

Comment:
Not needed (no assignment)

Reply:
That is technically correct, but I think this is good defensive coding / future-proofing.